### PR TITLE
task 4 submission ITS_One

### PR DESCRIPTION
**Pull request by abhinavkgrd**

### CODE REVIEW

**Review by sarthakgupta072 — COMMENTED** [review]
Great!

The extension is working fine.
Can you also upload the code here?

**Comment by abhinavkgrd** [comment]
Thanks.. I will upload the code in just a sec.

**Review by sarthakgupta072 — COMMENTED** [review]
Nice!!

It's ready to be merged. 

Can you also comment your code at appropriate places, so that others can understand it too, and use it as a reference?

**Comment by abhinavkgrd** [comment]
Ok....no problem

File: task-4/ITS_One/code/history_tracker/popup.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+$("#submit").click(function () {
```
Comment (sarthakgupta072) [inline task-4/ITS_One/code/history_tracker/popup.js:25]: Just comment this file.

Comment (abhinavkgrd) [reply task-4/ITS_One/code/history_tracker/popup.js:25]: done

**Review by sarthakgupta072 — APPROVED** [review]
Great! You are the first one to complete all the tasks. :)

Congratulations!!! 💥

**Comment by abhinavkgrd** [comment]
Thanks ..... it was a great learning experience

**Pull request merged**